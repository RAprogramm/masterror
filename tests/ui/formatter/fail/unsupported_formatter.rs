// SPDX-FileCopyrightText: 2025 RAprogramm <andrey.rozanov.vl@gmail.com>
//
// SPDX-License-Identifier: MIT

use masterror::Error;

#[derive(Debug, Error)]
#[error("{value:y}")]
struct UnsupportedFormatter {
    value: u32,
}

fn main() {}
