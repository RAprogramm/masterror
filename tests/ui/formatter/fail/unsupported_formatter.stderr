error: placeholder spanning bytes 0..9 uses an unsupported formatter
 --> tests/ui/formatter/fail/unsupported_formatter.rs:8:9
  |
8 | #[error("{value:y}")]
  |         ^^^^^^^^^^^
